### ISSUE TITLE
Method of undefined issues in new versions

### ISSUE THREAD

**Issue by arthurdenner** [samdenty/console-feed#62]
I'm investigating what was reported on #61 within CodeSandbox. I tried to upgrade to the newest version (3.1.9) and now we're getting `Cannot read property 'toUpperCase' of undefined` in some cases.

I think that instead of https://github.com/samdenty/console-feed/commit/9acd017d4d64dfdf875d1d5e5037c51d0fd5b1c5, we should check for `!error` in the `Error` component and return `The error you provided does not contain a stack trace.`.

The original issue that triggered my investigation is https://github.com/codesandbox/codesandbox-client/issues/4963 - we were getting a blank error message there, then we got `indexOf of undefined` and now `toUpperCase of undefined`.

What do you think about my suggestion? `The error you provided does not contain a stack trace.` is what we get in the browser console inside CodeSandbox, so the behaviour would match in our case I think. I was trying to create a reproducible example in the demo app to verify the behaviour but couldn't do it.

**Comment by samdenty** [comment]
@SaraVieira what did you do to fix this previously?

**Comment by samdenty** [comment]
https://github.com/samdenty/console-feed/commit/de41ef2662019c645a652199d9bdbbfc71419f5e should fix https://github.com/codesandbox/codesandbox-client/issues/4963

**Comment by arthurdenner** [comment]
Alright. When we get a new version I can try it out and see what happens.

**Comment by SaraVieira** [comment]
Yeah, this was an emotion issue, cleaning and running yarn again should fix it.

Make sure to delete the yarn file :/

**Comment by marmos91** [comment]
I am experiencing the same issue :(

**Comment by zdllucky** [comment]
I had this problem when passing incompatible log object. Some loggers today have key `level` instead of `method`, what was exactly my case.
